x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/rest/cat/RestCatJobsAction.java

```diff
@@ -183,6 +183,31 @@ protected Table getTableWithHeader(RestRequest request) {
             TableColumnAttributeBuilder.builder("number of bucket allocation failures", false)
                 .setAliases("mbaf", "modelBucketAllocationFailures")
                 .build());
+        table.addCell("model.categorization_status",
+            TableColumnAttributeBuilder.builder("current categorization status")
```

Comment:
since this is configured to be included by default, you will probably have to update the cat job yaml test

Reply:
Hmm, since only a small proportion of jobs use categorization I should probably change this one so it's not included by default.